internal/backends/qt/qt_accessible.rs

```diff
@@ -0,0 +1,699 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+// cspell:ignore descendents qobject qwidget
+
+use crate::accessible_generated::*;
+
+use i_slint_core::accessibility::AccessibleStringProperty;
+use i_slint_core::item_tree::{ItemRc, ItemWeak};
+use i_slint_core::properties::{PropertyChangeHandler, PropertyTracker};
+use i_slint_core::SharedVector;
+
+use cpp::*;
+use pin_project::pin_project;
+use qttypes::QString;
+
+use alloc::boxed::Box;
+use core::ffi::c_void;
+use std::pin::Pin;
+
+pub struct AccessibleItemPropertiesTracker {
+    accessible_item: *mut c_void,
+}
+
+// KEEP IN SYNC WITH CONSTANTS IN C++
+const NAME: u32 = QAccessible_Text_Name;
+const DESCRIPTION: u32 = QAccessible_Text_Description;
+const VALUE: u32 = QAccessible_Text_Value;
+const CHECKED: u32 = QAccessible_Text_UserText as u32;
+const VALUE_MINIMUM: u32 = CHECKED + 1;
+const VALUE_MAXIMUM: u32 = VALUE_MINIMUM + 1;
+const VALUE_STEP: u32 = VALUE_MAXIMUM + 1;
+
+impl PropertyChangeHandler for AccessibleItemPropertiesTracker {
+    fn notify(&self) {
+        let accessible_item = self.accessible_item;
+        cpp!(unsafe [accessible_item as "Slint_accessible_item*"] {
+            QTimer::singleShot(0, [accessible_item]() {
+                auto data = accessible_item->data();
```

Comment:
isn't it possible that accessible_item be destroyed there?  We could query the data and the object before calling singleShot.  we can even put the object in a QPointer.

(this is also valid for the other use of singleShot)

Reply:
The tree is very static at this time, so I do not think that is a problem right now. But using a QPointer is a good idea, I'll change that.

Reply:
Small implementation hurdle: QAccessibleInterface is no QObject. But not really a problem.